Source/WTF/WTF.xcodeproj/project.pbxproj

```diff
@@ -972,6 +973,20 @@
 			runOncePerArchitecture = 0;
 			script = "cp -f \"${INPUT_FILE_PATH}\" \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 		};
+		DD9C67F72D6FE2C600D4BEC7 /* PBXBuildRule */ = {
+			isa = PBXBuildRule;
+			compilerSpec = com.apple.compilers.proxy.script;
+			dependencyFile = "$(DERIVED_FILES_DIR)/$(INPUT_FILE_BASE).$(arch).d";
+			filePatterns = "*/generate-platform-args";
+			fileType = pattern.proxy;
+			inputFiles = (
+			);
```

Comment:
This isn't needed because there's a `dependencyFile`, correct?

Reply:
Exactly. The discovered dependency file will list wtf/Platform.h and all the headers transitively included.